paimon-format/src/main/java/org/apache/paimon/format/avro/AvroFileFormat.java

```diff
@@ -62,11 +62,14 @@ public class AvroFileFormat extends FileFormat {
     public static final ConfigOption<Map<String, String>> AVRO_ROW_NAME_MAPPING =
```

Comment:
In AvroFileFormat, IDENTIFIER && AVRO_OUTPUT_CODEC && AVRO_ROW_NAME_MAPPING should be private.